cmd/state-svc/internal/hash/file_hasher.go

```diff
@@ -24,11 +25,11 @@ type FileHasher struct {
 
 func NewFileHasher() *FileHasher {
 	return &FileHasher{
-		cache: cache.New(cache.NoExpiration, cache.NoExpiration),
+		cache: cache.New(cache.NoExpiration, 24*time.Hour),
 	}
 }
 
-func (fh *FileHasher) HashFiles(files []string) (string, error) {
+func (fh *FileHasher) HashFiles(files []string) (hash string, rerr error) {
```

Comment:
```suggestion
func (fh *FileHasher) HashFiles(files []string) (string, error) {
```

Doesn't look like naming these is used for anything. And it can actually cause bug so better to avoid.

Reply:
Forgot that we need the named error return for the deferred error handling https://github.com/ActiveState/cli/pull/3484/files#diff-f95330cc381d368f14952fbad1b24b3a714997bab5851fba341014a2271774d0R41

Reply:
ah I see. In that case I suggest renaming `hash` to `_`, so it doesn't clash with your code. You are using a `hash` var in the function body. I don't think it clashes as currently written, but it feels error-prone.